### DISCUSSION TITLE
How do I change the depot download path?

### DISCUSSION THREAD

**Discussion by Skyguy7878** [SteamRE/DepotDownloader#475] [Q&A]
I'm kind of having trouble right now because my C drive is really small and has like 5 gigs of storage free so it can't download the game I want it to, is there any way I can change the path that the depot downloads to so I can actually download and play my game (game I'm looking to play is Five Nights at Freddy's: Security Breach, app: 747660 depot: 747661 manifest: 1134117433438814396)?

**Comment by yaakov-h** [comment]
Does this help you, or is there a different path you want to change?

https://github.com/SteamRE/DepotDownloader/blob/master/README.md?plain=1#L48

**Reply by Skyguy7878** [reply]
Yes, it does! Thank you a lot! Time to go experience maximum glitches (why am I like this lol)!
